content/en/account_management/rbac/_index.md

```diff
@@ -59,11 +60,9 @@ To create a custom role:
 1. Go to your [Datadog Roles page][1].
 2. Select **New Role** in the upper right corner of the page.
 3. Give a name to your role.
-4. Optional - Assigning a set of permissions to your Role. See the [Datadog Role permissions][2] documentation for more information about available permissions.
+4. Assign a set of permissions to your role. See [Datadog Role Permissions][2] for more information about available permissions.
 
-{{< img src="account_management/rbac/create_role.png" alt="Create a custom Role" style="width:90%;">}}
-
-Once a role is created you can [add this role to existing users][3].
+Once a role is created, you can [add the role to existing users][3].
 
```

Comment:
Add a small blurb on Cloning & Role templates + Screenshots somewhere here. Something like - 
Alternatively Custom Roles can also be create by cloning existing roles or by applying Datadog prescribed templates as a starting point.